typescript/test/world-company-remuneration/basket-informations.test.ts

```diff
@@ -0,0 +1,16 @@
+import {BasketInformations} from "../../src/world-company-remuneration/basket-informations";
+
+describe("a basket should cost", () => {
```

Comment:
The header should be something like Testing BasketInformations.
So in the logs its easier to identify the failed test suite

Reply:
Hello, as the repo said, this is only a support for interviewing people, we already know that they are a lot of issues in this code. Thank you for your participation anyway